core/src/main/scala/org/broadinstitute/dsde/rawls/model/SpendReportingModel.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.rawls.model
+
+import org.broadinstitute.dsde.workbench.model.google.{BigQueryDatasetName, GoogleProject}
+import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport._
+import org.joda.time.DateTime
+import spray.json.DefaultJsonProtocol._
+
+case class BillingProjectSpendConfiguration(datasetGoogleProject: GoogleProject, datasetName: BigQueryDatasetName)
+
+case class BillingProjectSpendExport(billingProjectName: RawlsBillingProjectName, billingAccountId: RawlsBillingAccountName, spendExportTable: Option[String])
+
+case class SpendReportingResults(spendDetails: Seq[SpendReportingAggregation], spendSummary: SpendReportingForDateRange)
+case class SpendReportingAggregation(aggregationKey: SpendReportingAggregationKey, spendData: Seq[SpendReportingForDateRange])
+case class SpendReportingForDateRange(
+                                       cost: String,
+                                       credits: String,
+                                       currency: String,
+                                       startTime: DateTime,
```

Comment:
I feel like some documentation about the expectations around startTime/endTime would be helpful. Possibly also credits/cost.